geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/geoportal/vars.yaml

```
@@ -265,6 +265,10 @@ vars:
       Sincerely yours
       The GeoMapFish team
 
+  allowed_hosts:
+    # TODO: Add the allowed hosts for the application (themes and dynamic.json)
```

You'll do that in a second PR ?

---

No, it's a TODO for the scaffolded project :-)

---

the TODO makes it confusing.
```suggestion
    You must add here the allowed hosts for the application (themes and dynamic.json)
```